ckanext/who_afro/assets/css/userprofile.css

```diff
@@ -0,0 +1,345 @@
+.wrapper {
+    box-shadow: none;
+    border: 0;
+}
+
+.page-header {
+    background-color: white;
+}
+
+div.module-content {
+    border: var(--who-blue-65-percent) thin solid;
+    border-top: none;
+}
+
+header,
+header ul.nav.nav-tabs {
+    width: calc(100% + 1px);
+}
+
+header ul.nav.nav-tabs li:first-child {
+    width: 25%;
+}
+
+header ul.nav.nav-tabs li:not(:first-child) {
+    width: 24.3%;
+    margin-left: 0.7%;
+}
+
+header ul.nav.nav-tabs li:last-child {
+    width: 24.3%;
+    margin-left: 0.7%;
+}
+
+.page-header .nav-tabs li.active a, .page-header .nav-tabs a:hover {
+    border: var(--who-blue-65-percent) thin solid;
+    border-bottom-color: transparent;
+}
+
+.nav-tabs .nav-link, .page-header .nav-tabs li a,
+.nav-tabs .nav-link, .page-header .nav-tabs li a:hover {
+    background-color: var(--who-blue-ten-percent);
+}
+
+.nav-tabs,
+.nav-tabs .nav-link, .page-header .nav-tabs li a {
+    border-bottom-color: var(--who-blue-65-percent);
+}
+
+.nav.nav-tabs li a {
+    color: var(--primary-navy);
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 20px;
+    text-align: center;
+}
+
+@media only screen and (max-width: 510px) {
+    header ul.nav.nav-tabs li,
+    header ul.nav.nav-tabs li:first-child,
+    header ul.nav.nav-tabs li:not(:first-child),
+    header ul.nav.nav-tabs li:last-child {
+        width: 100%;
+    }
+    header ul.nav.nav-tabs li:first-child {
+        margin-left: 4px;
+    }
+    .nav.nav-tabs li a {
+        text-align: left;
+    }
+    .dashboard-add-team-btn {
+        width: 100% !important;
+        margin-top: 16px;
+        margin-bottom: 16px;
+    }
+
+}
+
+@media only screen and (max-width: 545px) {
+    .user-dashboard-main .pull-right {
+        float: none;
+        width: 100%;
+    }
+    .user-dashboard-main #followee-popover {
+        width: 100%;
+    }
+    .user-dashboard-main small.text-muted.fs-5 {
+        text-align: center;
+    }
+}
+
+
+.page-header.module-content {
+    padding: 0;
+    width: 100%;
+}
+
+.dashboard-organisation-main article.module,
+.user-dashboard-main article.module{
+    padding: 0;
+    margin-bottom: 115px;
+}
+
+.media-grid .media-heading {
+    margin: 8px 0 0;
+}
+
+.dashboard-organisation-main .media-grid {
+    margin-bottom: 32px;
+}
+
+.show-all-org,
+.show-all-org:hover,
+.show-all-pkg,
+.show-all-pkg:hover {
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 24px;
+    text-decoration: underline;
+}
+
+@media only screen and (max-width: 462px) {
+  #activity_page_buttons, #activity_page_buttons a.btn {
+    margin-top: 16px;
+    width: 100%;
+    }
+}
+
+/* header navbar */
+@media only screen and (min-width: 992px) {
+    .masthead .main-navbar ul li.active a {
+        color: var(--primary-navy) !important;
+    }
+
+    .masthead .main-navbar ul li:last-child a {
+        padding-bottom: 37px !important;
+        border-bottom: var(--secondary-who-blue) 3px solid;
+        border-radius: 0;
+        z-index: 0;
```

Comment:
Note that the border bottom here should actually be the standard way of identifying the active element in the main nav.  You have just implemented it for the user profile page, but it should be implemented for all main nav elements. So when you click on "Find Data" it should be marked as active with the bottom border as well. At the moment this is done through highlighting the text, but this isn't lewis' original design.  Would be good if you could clean this up and get the active nav element always marked with the blue bottom border for all pages.